src/CloudFlareZoneInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\cloudflare\CloudflareZoneInterface.
+ */
+
+namespace Drupal\cloudflare;
+use CloudFlarePhpSdk\ApiTypes\Zone\ZoneSettings;
+
+/**
+ * Zone methods for CloudFlare.
+ */
+interface CloudFlareZoneInterface {
+
+  /**
+   * Updates a zone's settings from CloudFlare's API.
+   *
+   * @param \CloudFlarePhpSdk\ApiTypes\Zone\ZoneSettings $zone_settings
+   *   The updated settings object.
+   */
+  public function updateZoneSettings(ZoneSettings $zone_settings);
```

Comment:
Doesn't ``ZoneSettings`` have a interface instead? That allows better mockability, yet dogmatism about this kind of thing isn't healthy either.